### PR TITLE
Remove pointer on resizable panel handle

### PR DESCRIPTION
**Pull request by richardhjtan**
Refer to https://linear.app/cardstack/issue/CS-7253/eliminate-pointer-on-resizable-panel-handle

### What is changing 
Remove the visual pointer element on the resize handle component.

----

#### Before

image

#### After

remove-visual-pointy-element

### CODE REVIEW

**Comment by lukemelia** [comment]
Hi @hockjootan , the intent of this task was not well-communicated in the Linear issue. The goal is unrelated to the cursor. It is to remove the visual pointy element from the resize handle.  
Screenshot 2024-10-01 at 11 03 05 AM

File: packages/boxel-ui/addon/src/components/resizable-panel-group/handle.gts [hunk 1]
```diff
@@ -90,11 +90,11 @@ export default class Handle extends Component<Signature> {
       }
 
       .resize-handle.horizontal {
-        cursor: col-resize;
+        cursor: unset;
```
Comment (habdelra) [inline packages/boxel-ui/addon/src/components/resizable-panel-group/handle.gts]: please let's not change the cursor

Comment (richardhjtan) [reply packages/boxel-ui/addon/src/components/resizable-panel-group/handle.gts]: Thank you @habdelra for the feedback, will revert the changes as the intention was to get rid of the visual pointer

**Comment by richardhjtan** [comment]
Thank you @lukemelia for the clarification, I have reverted the initial changes and amend according to requirement.

Hi @habdelra I have update the PR, please have a look again thanks!

**Comment by habdelra** [comment]
This is what i'm seeing in your branch. looks perfect!
image

**Comment by habdelra** [comment]
I see a bunch of percy differences. can you merge main into your branch and see if that clears up the percy issues?

**Comment by richardhjtan** [comment]
@habdelra yes I noticed there are some snapshots changed after the rebase. Each of them is highlighting about the pointy visual elements, and I have reviewed and approved the snapshot

**Review by habdelra — APPROVED** [review]

**Pull request merged**